main.py

```diff
@@ -1,10 +1,48 @@
 import init_django_orm  # noqa: F401
+import json
+from django.utils import timezone
+import datetime
+
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_name, player_data in players_data.items():
+        race_data = player_data.get("race", {})
```

Comment:
Extract the player's race from player_data; if the value is None, no action is taken. No need to put default value

Reply:
fix in all places